Makefile

```diff
@@ -201,6 +202,56 @@ run-test-multikueue-e2e-%: FORCE
 	@echo Running multikueue e2e for k8s ${K8S_VERSION}
 	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) GINKGO_ARGS="$(GINKGO_ARGS)" JOBSET_VERSION=$(JOBSET_VERSION) ./hack/multikueue-e2e-test.sh
 
+SCALABILITY_RUNNER := $(ARTIFACTS)/scalability-runner
```

Comment:
The file is getting long, the main Makefile is the entry point for most developers, so it would be nice to make it as clean as possible. Running scalability tests seems niche. We may delegate to a scripts in hack as we do for other things.

Reply:
- All the binaries we are using internal or external have their own target here.
- All the tests have their make file target, using hack helpers only if some complex setup is necessary.